docker/registry/nginx-noroot-base/gradle.properties

```diff
@@ -0,0 +1,3 @@
+io.deephaven.project.ProjectType=DOCKER_REGISTRY
+deephaven.registry.imageName=ghcr.io/deephaven/nginx-noroot-base:latest
+deephaven.registry.imageId=ghcr.io/deephaven/nginx-noroot-base
```

Comment:
TODO: update this with `@sha256:` once we get no-root image merged to deephaven-base-images